app/content/react/library-reference/widgets.md

```diff
@@ -58,9 +59,31 @@ Renders a `<CategoryWidget />` component, binded to a source at redux. The widge
       onError={console.error}
     />
   );
-
+  
   // The operationColumn wouldn't be required if using AggregationTypes.COUNT, to count the number of countries per continent
   ```
+  
+  In case you have the `population` disaggregated by gender: `population_m` and `population_f`, you can use an array in `operationColumn` and sum them into a single column using `joinOperation`. By this way, you avoid to have duplicated data in your dataset and you can compose widgets using multiple columns.
```

Comment:
Maybe "... and sum the values from the two columns using the `joinOperation` property`.